tests/parametric/test_config_consistency.py

```diff
@@ -43,3 +43,34 @@ def test_tracing_disabled(self, library_env, test_agent, test_library):
         assert (
             True
         ), "wait_for_num_traces raises an exception after waiting for 1 trace."  # wait_for_num_traces will throw an error if not received within 2 sec, so we expect to see an exception
+
+
+def set_service_version_tags():
+    env1 = {}
+    env2 = {"DD_SERVICE": "test_service", "DD_VERSION": "5.2.0"}
+    return parametrize("library_env", [env1, env2])
+
+
+@scenarios.parametric
+@features.tracing_configuration_consistency
+class Test_Config_UnifiedServiceTagging:
```

Comment:
Cool! Is the approach to contain all service, env and version test functions inside this class?

Reply:
Yes, as each individual environment setup should not affect the others.